src/components/top_bar/top_bar.ts

```diff
@@ -425,18 +509,31 @@ export class TopBar extends Component<Props, SpreadsheetChildEnv> {
     setStyle(this.env, { [target]: color });
   }
 
-  setBorder(command: BorderCommand) {
+  setBorder(property: String, value: String) {
+    switch (property) {
```

Comment:
avoid swtich/case when you can.
Here you can just create three different methods